<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1537</width>
    <height>898</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>785</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>⤬</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCollapse">
    <property name="geometry">
     <rect>
      <x>1442</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>—</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonNextPage">
    <property name="geometry">
     <rect>
      <x>1490</x>
      <y>800</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonPreviosPage">
    <property name="geometry">
     <rect>
      <x>1440</x>
      <y>800</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelNumberPage">
    <property name="geometry">
     <rect>
      <x>1392</x>
      <y>800</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0/0</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPrice_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPicture_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_1">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="buttonBasket">
    <property name="geometry">
     <rect>
      <x>1450</x>
      <y>90</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>🛒</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>140</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPicture_2">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>27</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPrice_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>140</y>
      <width>291</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPicture_3">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>27</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPrice_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>290</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPicture_4">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>37</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPrice_4">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>299</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_4">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>490</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPicture_5">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>37</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPrice_5">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>299</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_5">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>490</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <widget class="QLabel" name="productPicture_6">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>37</y>
       <width>291</width>
       <height>261</height>
      </rect>
     </property>
     <property name="text">
      <string>Picture</string>
     </property>
    </widget>
    <widget class="QLabel" name="productPrice_6">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>299</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Price</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonOpenWidget_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>291</width>
       <height>331</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonAddProduct_6">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>300</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>➕</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineSearch">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>1211</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>20</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>⨯</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSearchProduct">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>20</y>
      <width>31</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>🔍</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioPopularProduct">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>По популярности</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIncreasingName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>60</y>
      <width>301</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>По увеличению наименования товара</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioReductionName">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>311</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>По убыванию наименования товара</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioIncreasingPrice">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>60</y>
      <width>221</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>По увеличению цены</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioReductionPrice">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>60</y>
      <width>181</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>По убыванию цены</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxCategory">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>90</y>
      <width>621</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxSubcategoty">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>90</y>
      <width>621</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1537</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
